### PR TITLE
Add bitsong fantoken assetlist

### PR DESCRIPTION
**Pull request by angelorc**
This PR fix the `chain-info` with the new bitsong version and add the fantoken list

### CODE REVIEW

File: bitsong/assetlist.json [hunk 1]
```diff
@@ -22,7 +27,358 @@
...
+            "exponent": 0,
+            "aliases": ["ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09"]
+          }
+        ],
+        "base": "ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09",
+        "name": "Adam Clay FanToken",
+        "display": "clay",
+        "symbol": "CLAY",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09.png"
```
Comment (JeremyParish69) [inline bitsong/assetlist.json:55]: not required, but I'd recommend considering renaming these to something more human readable.

File: bitsong/assetlist.json [hunk 2]
```diff
@@ -22,7 +27,358 @@
...
+            "exponent": 0,
+            "aliases": ["ft25B30C386CDDEBD1413D5AE1180956AE9EB3B9F7"]
+          }
+        ],
+        "base": "ft25B30C386CDDEBD1413D5AE1180956AE9EB3B9F7",
+        "name": "Nicola Fasano Fantoken",
+        "display": "fasano",
+        "symbol": "FASANO",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bitsong/images/ft25B30C386CDDEBD1413D5AE1180956AE9EB3B9F7.png"
```
Comment (JeremyParish69) [inline bitsong/assetlist.json:80]: curious, will all these images still look appropriate if some app were to crop them to a circle, cutting off the corners?

File: bitsong/assetlist.json [hunk 3]
```diff
@@ -22,7 +27,358 @@
...
+          {
+            "denom": "ft2D8E7041556CE93E1EFD66C07C45D551A6AAAE09",
+            "exponent": 0
+          },
+          {
+            "denom": "clay",
+            "exponent": 6
+          },
+          {
+            "denom": "ibc/7ABF696369EFB3387DF22B6A24204459FE5EFD010220E8E5618DC49DB877047B",
```
[the remark this answers is not in the document]
Comment (angelorc) [reply bitsong/assetlist.json:45]: alright, will do

File: bitsong/assetlist.json (hunk 1 above)
Comment (angelorc) [reply bitsong/assetlist.json:55]: do you mean the logo_URIs? we prefer to set as denom, so it easily to get it

**Pull request merged**